content/integrations/wmi.md

```diff
@@ -1,15 +1,134 @@
+2.  Implement a COM-based WMI provider for your application. You would typically only do this if you are using a non-.NET language.
+
+To learn more about using System.Diagnostics, refer to [the MSDN documentation here](https://msdn.microsoft.com/en-us/library/system.diagnostics.performancecounter(v=vs.110).aspx). After adding your metric you should be able to find it in WMI. To browse the WMI namespaces you may find this tool useful: [WMI Explorer](https://wmie.codeplex.com/). You can find the same information with Powershell [here](https://msdn.microsoft.com/en-us/powershell/scripting/getting-started/cookbooks/getting-wmi-objects--get-wmiobject-). Also review the information in the [Datadog Knowledge Base article](https://help.datadoghq.com/hc/en-us/articles/205016075-How-to-retrieve-WMI-metrics).
+
+If you assign the new metric a category of My_New_Metric, the WMI path will be 
+`\\<ComputerName>\ROOT\CIMV2:Win32_PerfFormattedData_My_New_Metric`
+
+If the metric isn't showing up in WMI, try running `winmgmt /resyncperf` to force the computer to reregister the performance libraries with WMI.
+
+
+# Configuration
+
+1.  Click the **Install Integration** button on the WMI Integration Tile. 
+2.  Open the Datadog Agent Manager on the Windows server.
+3.  Edit the `Wmi Check` configuration.
+  
+        init_config:
+
+        instances:
+          - class: Win32_OperatingSystem
+            metrics:
+              - [NumberOfProcesses, system.proc.count, gauge]
+              - [NumberOfUsers, system.users.count, gauge]
+
+          - class: Win32_PerfFormattedData_PerfProc_Process
+            metrics:
+              - [ThreadCount, proc.threads.count, gauge]
+              - [VirtualBytes, proc.mem.virtual, gauge]
+              - [PercentProcessorTime, proc.cpu_pct, gauge]
+            tag_by: Name
+
+          - class: Win32_PerfFormattedData_PerfProc_Process
+            metrics:
+              - [IOReadBytesPerSec, proc.io.bytes_read, gauge]
+            tag_by: Name
+            tag_queries:
+              - [IDProcess, Win32_Process, Handle, CommandLine]
+
+    Note that the default configuration uses the filter clause to limit the metrics pulled. Either set the filters to valid values or remove them as shown above to collect the metrics.
+
+    The metrics definitions include three components: class property in WMI, metric name as it appears in Datadog, and the metric type. 
+
+4.  Restart the agent.
+
+## Configuration Options
+
+Each WMI query has 2 required options, `class` and `metrics` and six optional options, `host`, `namespace`, `filters`, `tag_by`, `constant_tags` and `tag_queries`.
+
+`host` is the optional target of the WMI query, `localhost` is assumed by default if you set this option, make sure that Remote Management is enabled on the target host [see here](https://technet.microsoft.com/en-us/library/Hh921475.aspx) for more information.
```

Comment:
I think this is missing punctuation (it's a bit of a run-on). How about:

`host` is the optional target of the WMI query and `localhost` is assumed by default if you set this option. Make sure that Remote Management is enabled on the target host. [See here](https://technet.microsoft.com/en-us/library/Hh921475.aspx) for more information.